inf-clojure.el

```diff
@@ -609,7 +609,9 @@ This should usually be a combination of `inf-clojure-prompt' and
   :package-version '(inf-clojure . "2.0.0"))
 
 (defcustom inf-clojure-auto-mode t
-  "When non-nil, automatically enable inf-clojure-minor-mode for all Clojure buffers."
+  "Automatically enable inf-clojure-minor-mode.
+All buffers will automatically be in `inf-clojure-minor-mode'
```

Comment:
The updated wording sounds a bit confusing to me. Perhaps you still want to mention this is for Clojure buffers.

Reply:
Good call. I wanted to fix it but emacs is quite annoying about the first line being short and then having a subsequent line. I had to make two duplicative sentences instead of just a single one because of like 3 characters. That and the two spaces after periods rule always annoy me. :)